resources/install.rb

```diff
@@ -20,15 +20,54 @@
 include IISCookbook::Helper
 
 property :source, String
-property :additional_components, Array, default: []
+property :additional_components, [Array, String],
```

Comment:
why should this now accept a string?

Reply:
If you only want to specify one additional feature, as many recipes do, it saved cluttering every recipe with code to make a single item array